config/clients/python/template/tornado/rest.mustache

```diff
@@ -0,0 +1,222 @@
+{{>partial_header}}
+
+import io
+import json
+import logging
+import re
+
+# python 2 and python 3 compatibility library
+from six.moves.urllib.parse import urlencode
+import tornado
+import tornado.gen
+from tornado import httpclient
```

Comment:
I see `tornado` and `six` being imported here, but not specified in the deps. Is this expected?

Reply:
This file is not used.  Will remove